ignite/src/test/java/org/apache/zeppelin/ignite/IgniteSqlInterpreterTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.zeppelin.ignite;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Properties;
+
+import org.apache.ignite.IgniteCache;
+import org.apache.ignite.configuration.CacheConfiguration;
+import org.apache.zeppelin.interpreter.InterpreterContext;
+import org.apache.zeppelin.interpreter.InterpreterResult;
+import org.apache.zeppelin.interpreter.InterpreterResult.Code;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class IgniteSqlInterpreterTest {
+
+  private IgniteInterpreter ignite;
+  private IgniteSqlInterpreter sql;
+  private InterpreterContext context;
+
+  @Before
+  public void setUp() {
+    Properties p = new Properties();
+    p.setProperty("url", "localhost:11211");
+    p.setProperty("ignite.clientMode", "true");
+    ignite = new IgniteInterpreter(p);
+    ignite.open();
+
+    sql = new IgniteSqlInterpreter(p);
+    sql.open();
+
+    context = new InterpreterContext(null, null, null, null, null, null, null);
+  }
+
+  @After
+  public void tearDown() {
+    sql.close();
+    ignite.close();
+  }
+
+  @Test
+  public void testSql() {
+    CacheConfiguration<Integer, Person> cacheConf = new CacheConfiguration<Integer, Person>();
+    cacheConf.setName(null); // use default cache
+
+    IgniteCache<Integer, Person> cache = ignite.getIgnite().cache("query");
+    cache.put(1, new Person("sun", 100));
+    cache.put(2, new Person("moon", 50));
+    InterpreterResult result = sql.interpret("select name, age from Person where age > 10", context);
```

Comment:
Test is failing, with error
```
org.apache.zeppelin.interpreter.InterpreterException: java.sql.SQLException: Client is invalid. Probably cache name is wrong.
	at org.apache.zeppelin.ignite.IgniteSqlInterpreter.open(IgniteSqlInterpreter.java:67)
	at org.apache.zeppelin.ignite.IgniteSqlInterpreterTest.setUp(IgniteSqlInterpreterTest.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
Caused by: java.sql.SQLException: Client is invalid. Probably cache name is wrong.
	at org.apache.ignite.internal.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.apache.ignite.IgniteJdbcDriver.connect(IgniteJdbcDriver.java:274)
	at org.apache.zeppelin.ignite.IgniteSqlInterpreter.open(IgniteSqlInterpreter.java:65)
	... 25 more
```

It is very appreciated if someone can give me advice a correct way of unittest sql for ignite.